src/java/org/apache/cassandra/index/sai/plan/StorageAttachedIndexSearcher.java

```diff
@@ -211,6 +211,8 @@ public UnfilteredRowIterator computeNext()
             skipToNextPartition();
 
             UnfilteredRowIterator iterator = nextRowIterator(this::nextSelectedKeyInRange);
+            logger.debug("Got row iterator: " + iterator);
```

Comment:
This will be printed in production (because we configure the logger at DEBUG level), let's remove this line and the ones below

Reply:
fixed